apps/designer/app/designer/features/props-panel/props-panel.tsx

```diff
@@ -35,20 +42,34 @@ const Property = ({
   const type = argType?.control.type || "text";
   const defaultValue = argType?.control.defaultValue;
   const options = argType?.options;
+  const allProps = meta.argTypes ? Object.keys(meta.argTypes) : [];
+
   return (
     <Grid
       gap="1"
       css={{ gridTemplateColumns: "1fr 1fr auto", alignItems: "center" }}
     >
-      <TextField
-        readOnly={required}
-        variant="ghost"
-        placeholder="Property"
+      <Combobox
         name="prop"
+        items={allProps}
         value={prop}
-        autoFocus={prop === ""}
-        onChange={(event) => {
-          onChange(id, "prop", event.target.value);
+        itemToString={(item) => item ?? ""}
+        onItemSelect={(value) => {
+          onChange(id, "prop", value);
+        }}
+        disclosure={({ inputProps, toggleProps }) => (
+          <ComboboxTextField
+            toggleProps={toggleProps}
+            inputProps={{
```

Comment:
Same as my next comment.

Reply:
going to rename disclosure to renderContent, renderTrigger and renderList

Reply:
wait, you meant this is related to some bug